nautobot/core/templates/nautobot_config.py.j2

```diff
@@ -168,7 +168,104 @@ INTERNAL_IPS = ("127.0.0.1", "::1")
 
 # Enable custom logging. Please see the Django documentation for detailed guidance on configuring custom logs:
 #   https://docs.djangoproject.com/en/stable/topics/logging/
-LOGGING = {}
+LOG_LEVEL = "DEBUG" if DEBUG else "INFO"
+DJANGO_LOG_LEVEL = "INFO"
+
+# Verbose logging during normal development operation, but quiet logging during unit test execution
+LOGGING = {
+    "version": 1,
+    "disable_existing_loggers": False,
+    "formatters": {
+        "normal": {
+            "format": "%(asctime)s.%(msecs)03d %(levelname)-7s %(name)s :\n  %(message)s",
+            "datefmt": "%H:%M:%S",
+        },
+        "verbose": {
+            "format": "%(asctime)s.%(msecs)03d %(levelname)-7s %(name)-20s %(filename)-15s %(funcName)30s() :\n  %(message)s",
+            "datefmt": "%H:%M:%S",
+        },
+    },
+    "handlers": {},
+    "loggers": {},
+    "root": {
+        "level": LOG_LEVEL,
+    },
+}
```

Comment:
This is a great default but it's apparently getting clobbered by the `if...else` below.